esp/templates/contact.html

```diff
@@ -26,13 +28,14 @@
   {% inline_qsd_block "contact_header_text" %}
   # Contact Us
 
-  Fill out this form to send a quick e-mail to the {{ settings.ORGANIZATION_SHORT_NAME }} office. We will get back to you as soon as possible (usually within a day). Submitting this form does not add you to our mailing list.
-
-  <small>(Unless of course, you, uh, ASK to be added to our mailing list.)</small>
+  Fill out this form to send a quick email to {{ settings.ORGANIZATION_SHORT_NAME }}.
+  We will get back to you as soon as possible (usually within a day).
+  Submitting this form does not add you to our mailing list.
 
   ## Note to Parents:
 
-  If your child is or wants to be a student at our programs, we'd rather hear from them directly. We really do appreciate your interest, but we're an all-volunteer organization so we sometimes take a while to respond to e-mail, and we have an obligation to answer mail from our students first. So, please have your son or daughter send us an e-mail!
+  If your child is or wants to be a student at one of our programs, while we do appreciate your interest, we'd rather hear from the children directly.
```

Comment:
should be "them" or "the child", since the first clause refers to a singular child.

Reply:
Thanks